src/truncate.jl

```diff
@@ -0,0 +1,92 @@
+"""
+    Base.truncate(basis::RBasis, n_truth::Int)
+
+Truncate the [`RBasis`](@ref) to `n_truth` snapshots.
+
+Note that `n_truth` does not amount to the dimension of the truncated basis,
+but the number of truth solves included in the basis, which can feature degenerate
+snapshots.
+"""
+function Base.truncate(basis::RBasis, n_truth::Int)
```

Comment:
Most of this is not yet tested.

Reply:
Yes, I will add some tests tomorrow, today I ran out of time :-)